api/jiekouapi.php

```diff
@@ -428,7 +429,6 @@ function updatetokentime($con,$token,$ip) {
 	
 	function post($con,$token,$bid,$ip,$attachs) {
 		$time=time();
-		echo '<capu>';

```

Comment:
Thanks for fixing this small bug. Just one small suggestion, you can send a different PR for this. As I have explained in other comments. It's not a good idea to bring too many (not really related) modifications in one PR.